src/pages/Merchant/Register/Register.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+
+import styles from "./Register.module.scss";
+
+import Button from "components/Button/Button";
+import InputControl from "components/InputControl/InputControl";
+
+function Register() {
+  const [errors, setErrors] = useState({});
+
+  const fnameRef = useRef();
+  const lnameRef = useRef();
+  const emailRef = useRef();
+  const mobileRef = useRef();
+  const passRef = useRef();
+  const confirmpassRef = useRef();
+
+  const validateForm = () => {
+    const dummyErrors = {};
+    if (fnameRef.current.value === "") {
+      dummyErrors.fname = "Enter first name";
+      setErrors(dummyErrors);
+    }
+
+    if (lnameRef.current.value === "") {
+      dummyErrors.lname = "Enter last name";
+      setErrors(dummyErrors);
+    }
+
+    if (mobileRef.current.value === "") {
+      dummyErrors.mobile = "Enter mobile number";
+      setErrors(dummyErrors);
+    } else {
+      if (
+        mobileRef.current.value.length !== 10 &&
+        !/^\d{10}$/.test(mobileRef.current.value)
+      ) {
+        dummyErrors.mobile = "enter valid mobile number";
+        setErrors(dummyErrors);
+      }
+    }
+
+    if (emailRef.current.value === "") {
+      dummyErrors.email = "Enter email";
+      setErrors(dummyErrors);
+    } else {
+      if (
+        !/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(
+          emailRef.current.value
+        )
+      ) {
+        dummyErrors.email = "Enter valid email";
+        setErrors(dummyErrors);
+      }
+    }
+
+    if (passRef.current.value === "") {
+      dummyErrors.password = "Enter password";
+      setErrors(dummyErrors);
+    } else {
+      if (passRef.current.value.length < 8) {
+        dummyErrors.password = "Must be at least 8 characters";
+        setErrors(dummyErrors);
+      }
+      if (passRef.current.value.search(/[a-z]/i) < 0) {
+        dummyErrors.password = "Must contain at least one letter";
+        setErrors(dummyErrors);
+      }
+      if (passRef.current.value.search(/[0-9]/) < 0) {
+        dummyErrors.password = "Must contain at least one digit";
+        setErrors(dummyErrors);
+      }
+    }
+
+    if (confirmpassRef.current.value === "") {
+      dummyErrors.confirmpass = "Confirm password";
+      setErrors(dummyErrors);
+    } else {
+      if (confirmpassRef.current.value !== passRef.current.value) {
+        dummyErrors.confirmpass = "passwords must be same";
+        setErrors(dummyErrors);
+      }
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    validateForm();
```

Comment:
same mistake as in login handleSubmit function

Reply:
done